backend/api/db/crud/notebook_event.py

```diff
@@ -0,0 +1,91 @@
+import json
+from datetime import datetime
+from uuid import UUID
+
+from asyncpg import Record
+from asyncpg.connection import Connection
+
+from api.db.models.notebook_event import (
+    EventFrom,
+    EventStatus,
+    NotebookEvent,
+    NotebookEventInsert,
+)
+
+
+def parse_notebook_event(record: Record) -> NotebookEvent:
+    return NotebookEvent(
+        notebook_id=record["notebook_id"],
+        event_date=record["event_date"],
+        creator_id=record["creator_id"],
+        event=json.loads(record["event"]),
+        event_type=record["event_type"],
+        id=record["id"],
+        creation_date=record["creation_date"],
+    )
+
+
+def create_notebook_event(
```

Comment:
je propose qu'on renomme en 
```suggestion
def create_notebook_event_payload(
```
ou
```suggestion
def create_notebook_event_data(
```